modules/oci-publish/00_mod.mk

```diff
@@ -0,0 +1,29 @@
+# Push names is equivilent to build_names, additional names can be added for 
+# pushing images that are not build with the oci-build module
+push_names ?=
+push_names += $(build_names)
+
+# Sometimes we need to push to one registry, but pull from another. This allows
+# that.
+image_registry_rewrite ?= 
+
+# Utilities for extracting the key and value from a foo=bar style line
+kv_key = $(word 1,$(subst =, ,$1))
+kv_value = $(word 2,$(subst =, ,$1))
+
+# Apply the image_registry_rewrite rules, if no rules match an image then the 
+# image name is not changed. Any rules that match will be applied.
+#
+# For example, if there was a rule vanity-domain.com=real-registry.com/foo
```

Comment:
Could you clarify here which of the two domains is the one that eg. would be used in your Helm chart?
Based on the description, it is not yet clear to me which is the registry for pushing and which is the one for writing.

Reply:
Ok, that makes a lot of sense, could you also put this example and info in the comments in this file?